### DISCUSSION TITLE
DataTable Cell Edit options object returns the same rowData after filtering

### DISCUSSION THREAD

**Discussion by Alediran** [primefaces/community#1757] [PrimeReact]
I'm finding myself dealing with an odd issue. In one of the DataTable instances I use in my application I have Cell Edit enabled for some columns. The initial data I pass to the DataTable works fine in Cell Edit mode, but if I apply any one of the filters I use and go to edit a row, the rowData object in the ColumnEditorOptions property is not properly refreshed and instead it passes the data for that rowIndex that belonged to the previous result.

I'm using the 10.6.3 version of PrimeReact, I've tested using the lazy property. The filtering is done in the Back End, so the object I receive back is new, no filtering happens on the grid. Selection mode is set to checkbox.

**Comment by melloware** [comment]
Probably a bug. If you create a StackBlitz reproducer then please open a GitHub issue with your reproducible test case.
